.prettierrc

```diff
@@ -1,6 +1,10 @@
 {
-	"trailingComma": "none",
-	"tabWidth": 2,
-	"semi": true,
-	"singleQuote": true
-}
\ No newline at end of file
+  "trailingComma": "none",
```

Comment:
Can we keep the formatting as it was here?

Reply:
ok